src/testkit/scala/tools/testkit/ReflectUtil.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Scala (https://www.scala-lang.org)
+ *
+ * Copyright EPFL and Lightbend, Inc.
+ *
+ * Licensed under Apache License 2.0
+ * (http://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package scala.tools.testkit
+
+import scala.reflect.ClassTag
+
```

Comment:
```suggestion
import scala.util.chaining._
import java.lang.reflect.{Array => _, _}
```

Reply:
Right. Sorry I didn't test this properly locally. This is fixed now.